acquia/cli
# Fixes #697: Introduce ide:service-start and ide:service-stop

**Motivation**

Fixes #697 

**Proposed changes**
This PR introduces `supervisorctl` replacement for Cloud IDE so customers don't have to learn about it and use ACLI instead. In more details, we're introducing `ide:service-start` and `ide:service-stop` to manage services altogether.

**Alternatives considered**
N/A

**Testing steps**
Run `acli ide:service-stop` and `acli ide:service-start` against the following services: `php`, `apache`, `mysql` and confirm services are actually started or stopped.

```
test:~/project (pipelines-build-master *) $ ./acli.phar | grep service
  ide:service-restart               Restart a service in the Cloud IDE
  ide:service-start                 Start a service in the Cloud IDE
  ide:service-stop                  Stop a service in the Cloud IDE
test:~/project (pipelines-build-master *) $ for i in php mysql apache ; do ./acli.phar ide:service-stop $i ; done
Stopping php...
Stopped php
Stopping mysql...
Stopped mysql
Stopping apache...
Stopped apache
test:~/project (pipelines-build-master *) $ for i in php mysql apache ; do ./acli.phar ide:service-start $i ; done
Starting php...
Started php
Starting mysql...
Started mysql
Starting apache...
Started apache
```

1. Follow the [contribution guide](https://github.com/acquia/cli/blob/master/CONTRIBUTING.md#building-and-testing) to set up your development environment.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. (add specific steps for this pr)

**Merge requirements**
